### PR TITLE
[WFLY-11204] Upgrade Apache CXF to 5.2.7.Final

### PR DESCRIPTION
**Pull request by spyrkob**
Issue: https://issues.jboss.org/browse/WFLY-11204

CXF also upgraded snakeyaml to 1.23, but as far as I can tell it's a test dependency.

### CODE REVIEW

**Comment by bstansberry** [comment]
@asoldano or @jimma Please review.

**Review by jimma — APPROVED** [review]

**Comment by bstansberry** [comment]
@spyrkob @jimma  The Windows job had a number of WS related failures.

**Comment by spyrkob** [comment]
that looks related to a https://github.com/jboss/cxf/commit/0eb0d288d9cfb75f65f307a785707d522904d4f2
@asoldano - I guess we do need that fix after all

**Review by asoldano — COMMENTED** [review]
@bstansberry do you think supporting Early Access jdk11 version now that the final is available is a requirement? If so, we need to re-fork cxf from 3.2.7 tag.

**Comment by bstansberry** [comment]
@asoldano No it is not a requirement. AIUI our CI was just out of date.

**Comment by spyrkob** [comment]
@bstansberry do we need to update CI before this can be merged?

**Comment by bstansberry** [comment]
@spyrkob Yes. Ken's been waiting for the CI job queue to calm down before doing that, but that hasn't happened as there's a lot going as we're doing 16.0.0.Beta1 today.

If this is critical for the beta please let me know ASAP. I definitely wanted it in as it's better to do upgrades of major components earlier. But it's a micro update so it's not critical enough that I would tell Ken to disrupt things but updating CI immediately.

@asoldano @kabir FYI.

**Comment by luck3y** [comment]
retest this please

**Pull request merged**
